ports/mimxrt/main.c

```diff
@@ -106,6 +113,15 @@ int main(void) {
         #if MICROPY_PY_NETWORK
         mod_network_init();
         #endif
+        #if MICROPY_PY_BLUETOOTH
+        mp_bluetooth_hci_init();
+        #endif
+
+        #if MICROPY_PY_LWIP
+        // mod_network_lwip_init();
+        systick_enable_dispatch(SYSTICK_DISPATCH_LWIP, mod_network_lwip_poll_wrapper);
+        #endif
```

Comment:
Why disable/enable the dispatch on soft-reboot ?

Reply:
The idea was to ensure, that during shutdown no events are handled.